ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_linalg.py

```diff
@@ -657,3 +657,37 @@ def test_paddle_cond(*, dtype_and_x, p, on_device, fn_tree, frontend, test_flags
         atol=1e-5,
         p=p,
     )
+
+# dot
+@handle_frontend_test(
+    fn_tree="paddle.tensor.linalg.dot", 
+    dtype_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
Please use `valid` instead

```suggestion
        available_dtypes=helpers.get_dtypes("valid"),
```

Reply:
Gladly but the documentation in paddle specified that it uses only float, are you sure we should make that change?

Reply:
I don't think valid is the right option here since the "dot" function only works with float in paddle, also using valid doesn't make sense to me. Can you elaborate on how this is the better option?

Reply:
`helpers.get_dtypes("valid")` should resolve valid dtypes automatically, which in turn are specified by `@with_supported_dtypes`/`@with_unsupported_dtype` decorators.